Find a functionally equivalent code:android.media.AudioManager.unregisterMediaButtonIntent:COMMENT
Method Modifier: public      hidden      
Comment:/**
 * @hide
 */

@@ -1,8 +1,4 @@
 {
-    IAudioService service = getService();
-    try {
-        service.unregisterMediaButtonIntent(pi);
-    } catch (RemoteException e) {
-        Log.e(TAG, "Dead object in unregisterMediaButtonIntent" + e);
-    }
+    MediaSessionLegacyHelper helper = MediaSessionLegacyHelper.getHelper(mContext);
+    helper.removeMediaButtonListener(pi);
 }

